backend/spec/requests/api/v1/bugs_spec.rb

```
@@ -0,0 +1,75 @@
+        etc: "",
+        is_solved: false,
+        status: "draft",
+        environments: [],
+        attempts: [],
+        references: [],
+      }
+    end
+
+    context "ログインしている場合" do
+      it "正しい値が入力された場合、バグの登録が成功する" do
+        expect {
```

RuboCop (1.73)

[convention] Avoid using `{...}` for multi-line blocks.

(Style/BlockDelimiters)